### ISSUE TITLE
Feature Request: Pending Reboot

### ISSUE THREAD

**Issue by MisterPhoton** [sleevezipper/hass-workstation-service#55]
Hi, great work on this, looking forward to getting it up and running on my workstation.

Something I'd love to see added is a Pending Reboot sensor - would be nice to be able to trigger a reboot if, for instance. (pending reboot + workstation idle + weekend + night time hours).

**Comment by sleevezipper** [comment]
Thanks for the kind words! 

I'm not entirely sure what you mean by Pending Reboot. Do you mean what happens after Windows Update has downloaded a new update?

**Comment by MisterPhoton** [comment]
Yes - there are registry flags that are set by Windows after either a Windows Update or other change that requires a restart.

https://stackoverflow.com/questions/43596427/how-to-check-if-a-server-needs-a-reboot

The RMM tool we use at my office has a flag for it so I can bug people to restart their computer after an update. 😄

**Comment by sleevezipper** [comment]
Oh sweet! Thanks for the extra information.
